src/python/oftest/dataplane.py

```diff
@@ -16,7 +16,7 @@
 
 import sys
 import os
-import socket
+import socket, errno
```

Comment:
errno doesn't seem to be used in this file, so this change looks unnecessary

Reply:
error contains the errno.ENETDOWN which will be used later.

Reply:
if it's going to be used later, that line should be introduced with the code that actually uses it.
also, let's have each package imported on its own separate line